### PR TITLE
Setup development mode

### PR DESCRIPTION
**Pull request by kartikcho**
Ported https://github.com/bundly/dash/pull/50 to here, i.e the beta repo.

This PR configures `env.js` to help in dashboard frontend dev environment setup. In the future we should document on steps or procedure for setting up this dev environment for new contributors.

### CODE REVIEW

File: env.js [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 export let hosts = [];
 export let callbackUrl = '';
 export const TURN_INTERVAL = 0;
 if (process.env.NODE_ENV === 'production') {
     // React App Deployed URLs
     hosts = ['https://bundly.tech', 'https://dash-delta.vercel.app', 'https://dash.bundly.tech'];
     callbackUrl = 'https://bundly.tech/api';
-    DB_URL = 'mongodb://mongo:27017/dash'
-  } else {
+    DB_URL = 'mongodb://mongo:27017/dash';
```
Comment (sauravhiremath) [inline env.js:16]:
```suggestion
    DB_URL;
```

**Review by sauravhiremath — CHANGES_REQUESTED** [review]
LGTM. Just a minor change

**Comment by kartikcho** [comment]
@sauravhiremath done!

File: env.js [hunk 2]
```diff
@@ -1,22 +1,22 @@
 import dotenv from 'dotenv';
 dotenv.config();
 
-export const { DB_NAME, MONGO_PASSWORD, API_KEY, SALT_ROUNDS } = process.env;
+export const { MONGO_URI, API_KEY, SALT_ROUNDS } = process.env;
 
 export const API_PORT = process.env.API_PORT || 5000;
 
-export let DB_URL = '';
+export let DB_URL = MONGO_URI;
```
Comment (sauravhiremath) [inline env.js]:
```suggestion
export let DB_URL;
```

File: env.js [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 export let hosts = [];
 export let callbackUrl = '';
 export const TURN_INTERVAL = 0;
 if (process.env.NODE_ENV === 'production') {
     // React App Deployed URLs
     hosts = ['https://bundly.tech', 'https://dash-delta.vercel.app', 'https://dash.bundly.tech'];
     callbackUrl = 'https://bundly.tech/api';
-    DB_URL = 'mongodb://mongo:27017/dash'
-  } else {
+    DB_URL;
```
Comment (sauravhiremath) [inline env.js]:
```suggestion
    DB_URL = 'mongodb://localhost:27017/dash';
```

**Review by sauravhiremath — CHANGES_REQUESTED** [review]
My bad. Either me or GitHub (definitely me) messed up. Suggesting change again

**Comment by kartikcho** [comment]
Sorry for overlooking it before, we're all done now!

File: env.js [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
 export let hosts = [];
 export let callbackUrl = '';
 export const TURN_INTERVAL = 0;
 if (process.env.NODE_ENV === 'production') {
     // React App Deployed URLs
     hosts = ['https://bundly.tech', 'https://dash-delta.vercel.app', 'https://dash.bundly.tech'];
     callbackUrl = 'https://bundly.tech/api';
-    DB_URL = 'mongodb://mongo:27017/dash'
-  } else {
+    DB_URL = 'mongodb://localhost:27017/dash';
```
Comment (sauravhiremath) [inline env.js]:
```suggestion
    DB_URL = 'mongodb://mongo:27017/dash';
```

We using a docker image for running mongo, hence the URL

**Review by sauravhiremath — CHANGES_REQUESTED** [review]

**Pull request merged**